test/models/tf_ov_model_zoo/run_infer_multiple.sh

```diff
@@ -24,11 +24,32 @@ while getopts “:m:b:d:h” opt; do
 done
 shift $((OPTIND-1))
 
+# Display link in BuildKite CI Web
+function inline_link {
+  LINK=$(printf "url='%s'" "$1")
+  if [ $# -gt 1 ]; then
+    LINK=$(printf "$LINK;content='%s'" "$2")
+  fi
+  printf '\033]1339;%s\a\n' "$LINK"
+}
+
+# Display image in BuildKite CI Web
+function inline_image {
+  printf '\033]1338;url='"$1"';alt='"$2"'\a\n'
+}
+
+#==============================================================================
+#==============================================================================
+
 [ -f "$MANIFEST" ] || ( echo "Manifest not found: $MANIFEST !"; exit 1 )
 MANIFEST="$(cd "$(dirname "$MANIFEST")"; pwd)/$(basename "$MANIFEST")" # absolute path
 
 cd ${WORKDIR} || ( echo "Not found: $WORKDIR !"; exit 1 )
 echo "Dir: ${WORKDIR}"
+CSVFILE=benchmark_avg_infer_msec.csv
+[ -f "$CSVFILE" ] && rm $CSVFILE
```

Comment:
Typically "rm -f $CSVFILE" works for when you want to delete a file if it exists.